homekit/uuidgen.go

```diff
@@ -0,0 +1,20 @@
+package homekit
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/google/uuid"
+)
+
+func GetUUID(deviceName string) uuid.UUID {
+	uuidStore := fmt.Sprintf("./.uuid%s", deviceName)
```

Comment:
Could you use « %s/uuid.cfg » as there is already a directory with device information